src/V3LinkLValue.cpp

```diff
@@ -311,6 +311,7 @@ class LinkLValueVisitor final : public VNVisitor {
         }
     }
     void visit(AstNodePreSel* nodep) override {
+        if (AstSelBit* selbitp = VN_CAST(nodep, SelBit)) selbitp->access(m_setRefLvalue);
```

Comment:
But, if there's a select where the bit-to-select has itself a select, won't this break?  Please add a test e.g. `foo[sel_bit[3:2]] = bar`.

Nit:
```suggestion
        if (AstSelBit* const selbitp = VN_CAST(nodep, SelBit)) selbitp->access(m_setRefLvalue);
```

Reply:
Done.